### ISSUE TITLE
Exception when running `how_is duckinator/how_is` from master

### ISSUE THREAD

**Issue by indirect** [duckinator/how_is#16]
```
$ bundle exec how_is duckinator/how_is --report report.csv
bundler: failed to load command: how_is (/Users/andre.arko/workspace/duckinator/how_is/.bundle/ruby/2.3.0/bin/how_is)
NoMethodError: undefined method `/' for nil:NilClass
  /Users/andre.arko/workspace/duckinator/how_is/lib/how_is/analyzer.rb:92:in `average_age_for'
  /Users/andre.arko/workspace/duckinator/how_is/lib/how_is/analyzer.rb:36:in `call'
  /Users/andre.arko/workspace/duckinator/how_is/.bundle/ruby/2.3.0/gems/contracts-0.14.0/lib/contracts/method_reference.rb:43:in `send_to'
  /Users/andre.arko/workspace/duckinator/how_is/.bundle/ruby/2.3.0/gems/contracts-0.14.0/lib/contracts/call_with.rb:76:in `call_with'
  /Users/andre.arko/workspace/duckinator/how_is/.bundle/ruby/2.3.0/gems/contracts-0.14.0/lib/contracts/method_handler.rb:138:in `block in redefine_method'
  /Users/andre.arko/workspace/duckinator/how_is/lib/how_is.rb:41:in `generate_analysis'
  /Users/andre.arko/workspace/duckinator/how_is/.bundle/ruby/2.3.0/gems/contracts-0.14.0/lib/contracts/method_reference.rb:43:in `send_to'
  /Users/andre.arko/workspace/duckinator/how_is/.bundle/ruby/2.3.0/gems/contracts-0.14.0/lib/contracts/call_with.rb:76:in `call_with'
  /Users/andre.arko/workspace/duckinator/how_is/.bundle/ruby/2.3.0/gems/contracts-0.14.0/lib/contracts/method_handler.rb:138:in `block in redefine_method'
  /Users/andre.arko/workspace/duckinator/how_is/lib/how_is.rb:14:in `generate_report_file'
  /Users/andre.arko/workspace/duckinator/how_is/exe/how_is:52:in `<top (required)>'
  /Users/andre.arko/workspace/duckinator/how_is/.bundle/ruby/2.3.0/bin/how_is:22:in `load'
  /Users/andre.arko/workspace/duckinator/how_is/.bundle/ruby/2.3.0/bin/how_is:22:in `<top (required)>'
```

**Comment by duckinator** [comment]
The error is because it only supports PDF, JSON, or (with the next release) HTML exports. Leaving open because it should fail both earlier and more gracefully than that.

**Comment by duckinator** [comment]
Fixed in v3.0.
